src/main/scala/wdl4s/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -349,12 +349,21 @@ class WdlStandardLibraryFunctionsType extends WdlFunctions[WdlType] {
   def write_objects(params: Seq[Try[WdlType]]): Try[WdlType] = Success(WdlFileType)
   def write_json(params: Seq[Try[WdlType]]): Try[WdlType] = Success(WdlFileType)
   def glob(params: Seq[Try[WdlType]]): Try[WdlType] = Success(WdlArrayType(WdlFileType))
-  def size(params: Seq[Try[WdlType]]): Try[WdlType] = Success(WdlFloatType)
+  def isGoodFirstSizeParam(wdlType: WdlType): Boolean = wdlType match {
```

Comment:
nitpick, but while i kinda understand why all of the engine functions are crammed together like this, cramming multiline helper functions in there seems less aesthetically pleasing to me.

Also, is this intended to be overridden or something? If not maybe make it private?